modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalatraServerCodegen.java

```diff
@@ -57,7 +57,7 @@ public ScalatraServerCodegen() {
         apiPackage = "org.openapitools.server.api";
         modelPackage = "org.openapitools.server.model";
 
-        defaultIncludes = new HashSet<String>(
```

Comment:
Replace with `Sets.newHashSet(...`

Reply:
If you mean the guava Sets.newHashSet() that's considered deprecated since java 7:

https://guava.dev/releases/22.0/api/docs/com/google/common/collect/Sets.html#newHashSet--

> Note for Java 7 and later: this method is now unnecessary and should be treated as deprecated. Instead, use the HashSet constructor directly, taking advantage of the new "diamond" syntax.

Reply:
Done.